.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "cairo"]
```

Comment:
We should pin this to the latest stable version of Cairo, which is [2.8.4](https://github.com/starkware-libs/cairo/releases/tag/v2.8.4). This currently appears to be pinned to a random commit, which is _not_ what we want. We only want to integrate with an actual release of Cairo, so please fix this.

Reply:
@wzmuda I can see the link to be to commit [`f67829f8`](https://github.com/starkware-libs/cairo/commit/f67829f80d0b13bd4da466f07006b64fe421c42d). Am I misunderstanding anything?

![image](https://github.com/user-attachments/assets/929fa831-45c7-44ca-89de-0ee55a59aaa5)